linker/jvm/src/main/scala/org/scalajs/linker/PathIRContainer.scala

```diff
@@ -58,7 +58,7 @@ object PathIRContainer {
 
       blocking {
         // Open zip/jar file as filesystem.
-        val fs = FileSystems.newFileSystem(path, null)
+        val fs = FileSystems.newFileSystem(path, null: ClassLoader)
```

Comment:
Could you add a comment why the disambiguation is needed? Since we don't test on JDK 13, we might otherwise forget to preserve this ascription if we refactor that code.